<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>466</width>
    <height>234</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="open">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>92</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Open NARC</string>
    </property>
   </widget>
   <widget class="QComboBox" name="selectFile">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>10</y>
      <width>171</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>40</y>
      <width>261</width>
      <height>101</height>
     </rect>
    </property>
    <property name="title">
     <string>File Info</string>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>62</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string>File Size</string>
     </property>
    </widget>
    <widget class="QLabel" name="filesize">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>50</y>
       <width>71</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>91</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string>File Header</string>
     </property>
    </widget>
    <widget class="QLabel" name="fileheader">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>30</y>
       <width>71</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>91</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string>File Index</string>
     </property>
    </widget>
    <widget class="QLabel" name="fileindex">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>70</y>
       <width>71</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
    <widget class="QPushButton" name="replace">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>30</y>
       <width>92</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>Replace File</string>
     </property>
    </widget>
    <widget class="QPushButton" name="deleteFile">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>60</y>
       <width>92</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>Delete File</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>191</width>
      <height>101</height>
     </rect>
    </property>
    <property name="title">
     <string>Archive Info</string>
    </property>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>101</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string>Archive Size</string>
     </property>
    </widget>
    <widget class="QLabel" name="archivesize">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>50</y>
       <width>71</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>101</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string>Archive Name</string>
     </property>
    </widget>
    <widget class="QLabel" name="archivename">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>30</y>
       <width>71</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>101</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string>Total Files</string>
     </property>
    </widget>
    <widget class="QLabel" name="archivefiles">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>70</y>
       <width>81</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="addFile">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>150</y>
      <width>111</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Add File</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>466</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File...</string>
    </property>
    <addaction name="actionOpen"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionOpen">
   <property name="text">
    <string>Open...</string>
   </property>
  </action>
  <action name="actionReplace">
   <property name="text">
    <string>replace</string>
   </property>
  </action>
  <action name="actionDelete">
   <property name="text">
    <string>delete</string>
   </property>
  </action>
  <action name="actionAdd">
   <property name="text">
    <string>add</string>
   </property>
  </action>
  <action name="actionSelectFile">
   <property name="text">
    <string>selectFile</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>open</sender>
   <signal>clicked()</signal>
   <receiver>actionOpen</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>55</x>
     <y>48</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>replace</sender>
   <signal>clicked()</signal>
   <receiver>actionReplace</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>415</x>
     <y>108</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>deleteFile</sender>
   <signal>clicked()</signal>
   <receiver>actionDelete</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>415</x>
     <y>138</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>addFile</sender>
   <signal>clicked()</signal>
   <receiver>actionAdd</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>255</x>
     <y>188</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>selectFile</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>actionSelectFile</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>245</x>
     <y>48</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
